jetty-util/src/main/java/org/eclipse/jetty/util/component/ContainerLifeCycle.java

```diff
@@ -106,18 +106,20 @@ protected void doStart() throws Exception
                     switch (b._managed)
                     {
                         case MANAGED:
-                            if (!l.isRunning())
+                            if (l.isStopped())
```

Comment:
Looking at the state table from @joakime , I think we also need to start here if the bean is `FAILED`.  It is only if it is in the transit state of `STOPPING` that we should no longer start it.